### ISSUE TITLE
MProcess getUsingJavaClass does not set parameters properly.

### ISSUE THREAD

**Issue by mckayERP** [adempiere/adempiere#363]
The query is defined but parameters are not set.  Also, the className variable should be set using .getCanonicalName() function directly as getClass().getCononicalName() will always return "java.lang.class".

**Comment by yamelsenih** [comment]
Hi @mckayERP is nice to have a unique class for handle the process run, currently exists ProcessUtil, ProcessCtl, Procesa Builder and it is very hard when you want make a refactoring or add new functionality or validations.

**Comment by mckayERP** [comment]
Agree.  This issue is just that I like being able to use the class name
rather than the process ID which is difficult to find and verify.

So instead of

    ProcessInfo processInfo = ProcessBuilder.create(ctx)
                .process(53173)...

It is much more readable to do this

    ProcessInfo processInfo = ProcessBuilder.create(context)
            .process(org.compiere.process.MigrationFromXML.class)...
